.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our contributor guidelines: https://github.com/delta-incubator/delta-kernel-rs/blob/main/CONTRIBUTING.md
+  2. Ensure you have added or run the appropriate tests for your PR
```

Comment:
```suggestion
  2. Ensure you have added or run the appropriate tests for your PR and run `cargo fmt`.
```

Reply:
probably also include high-level testing command? just say to run `cargo t --workspace --all-features` or something?